pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -85,15 +89,15 @@ def solve_main(solve_data, config, fp=False, regularization_problem=False):
             update_suboptimal_dual_bound(solve_data, main_mip_results)
         if regularization_problem:
             config.logger.info(solve_data.log_formatter.format(solve_data.mip_iter, 'Reg '+solve_data.regularization_mip_type,
-                                                               value(
-                                                                   solve_data.mip.MindtPy_utils.loa_proj_mip_obj),
+                                                               value(solve_data.mip.MindtPy_utils.loa_proj_mip_obj),
                                                                solve_data.primal_bound, solve_data.dual_bound, solve_data.rel_gap,
                                                                get_main_elapsed_time(solve_data.timing)))
 
     elif main_mip_results.solver.termination_condition is tc.infeasibleOrUnbounded:
         # Linear solvers will sometimes tell me that it's infeasible or
         # unbounded during presolve, but fails to distinguish. We need to
         # resolve with a solver option flag on.
+        # TODO: appsi_cplex will fail here. Need to be fixed. test_mindtpy_feas_pump.py  test_mindtpy_global.py
```

Comment:
We should add this to an issue such that we keep track of it, and then reference the issue here in case we want to merge this PR before it is fixed

Reply:
Fixed